drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -255,6 +255,9 @@ static void nrf5_get_capabilities_at_boot(void)
 		((caps & NRF_802154_CAPABILITY_SECURITY) ? IEEE802154_HW_TX_SEC : 0UL)
 #if defined(CONFIG_IEEE802154_NRF5_MULTIPLE_CCA)
 		| IEEE802154_OPENTHREAD_HW_MULTIPLE_CCA
+#endif
+#if defined(CONFIG_IEEE802154_NRF5_CST_ENDPOINT)
+		| IEE802154_OPENTHREAD_HW_CST
```

Comment:
This will introduce build errors when both `CONFIG_IEEE802154_NRF5_MULTIPLE_CCA` and `CONFIG_IEEE802154_NRF5_CST_ENDPOINT` are enabled.

What about this?
```
#if defined(CONFIG_IEEE802154_NRF5_MULTIPLE_CCA)
		nrf5_data.capabilities |= IEEE802154_OPENTHREAD_HW_MULTIPLE_CCA
#endif
#if defined(CONFIG_IEEE802154_NRF5_CST_ENDPOINT)
		nrf5_data.capabilities |= IEE802154_OPENTHREAD_HW_CST
#endif
```

Reply:
There was a typo IEE802 ---> IEEE802 ...